aggregate/__main__.py

```diff
@@ -219,12 +282,123 @@ def create_cloud_function(
     # Slack notifications
     filter_string = fxn.name.apply(
         lambda fxn_name: f"""
-            resource.type="cloud_function"
-            AND resource.labels.function_name="{fxn_name}"
+            resource.type='cloud_function'
+            AND resource.labels.function_name='{fxn_name}'
+            AND severity >= WARNING
+        """
+    )
+
+    alert_policy = create_alert_policy(
+        name, fxn, 'function', filter_string, slack_channel
+    )
+
+    pulumi.export(f'{name}_function_name', fxn.name)
+
+    return fxn, trigger, alert_policy
+
+
+def create_cloudrun_job(
+    name: str = '',
+    config: dict = None,
+    service_account: str = None,
+    pubsub_topic: gcp.pubsub.Topic = None,
+    cloudrun_service: gcp.projects.Service = None,
+    slack_channel: gcp.monitoring.NotificationChannel = None,
+    prebuilt_images: dict[str, docker.Image] = None,
+):
+    """
+    Create a single Cloud Run Job. Include the pubsub trigger and event alerts
+    """
+
+    # Get prebuilt image, or build the new one
+    image_name = config['DOCKERIMAGE']
+    image: docker.Image = None
+    if image_name in prebuilt_images.keys():
+        image, full_image_name = prebuilt_images[image_name]
+    else:
+        image, full_image_name = setup_docker_image(
+            config['DOCKERIMAGE'], config['DOCKERFILE'], config['PROJECT']
+        )
+
+    # Create the Cloud Function
+    envs = [
+        {
+            'name': 'GCP_AGGREGATE_DEST_TABLE',
+            'value': config['GCP_AGGREGATE_DEST_TABLE'],
+        }
+    ]
+
+    container = gcp.cloudrun.ServiceTemplateSpecContainerArgs(
+        image=full_image_name,
+        envs=envs,
+    )
+
+    fxn = gcp.cloudrun.Service(
```

Comment:
Maybe Pulumi doesn't have it yet, but there should be CloudRun jobs, this avoid you running a flask server to respond to requests, you can't take any parameters I think, but that shouldn't matter for our use case. 

Eg, for seqr we update HPO terms once per week, this is what the UI for that looks like:

<img width="750" alt="image" src="https://user-images.githubusercontent.com/22381693/210442870-55d829d3-451e-4c04-932f-acce1b140eea.png">